django/contrib/admin/utils.py

```diff
@@ -481,23 +481,3 @@ def remove_trailing_data_field(fields):
     except NotRelationField:
         fields = fields[:-1]
     return fields
-
-
-def get_limit_choices_to_from_path(model, path):
```

Comment:
Actually, there's a comment right above where we use it that says "I don't know if this will ever do anything", so we might be ok :)